.github/workflows/deployment.yaml

```
@@ -0,0 +1,52 @@
+name: Deploy DEV/PRO
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - feat/webdev-47-migration-to-GH
+  workflow_dispatch:
+    inputs:
+      env:
+        description: 'Env to deploy'
+        required: true
+        default: 'dev'
+        type: choice
+        options:
+          - dev
+          - prod
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+env:
+  GH_TOKEN: ${{ secrets.SUPPORT_TOKEN }}
+  ENV_NAME: "${{ (inputs.env == 'prod' || github.event.schedule) && 'prod' || 'dev' }}"
+  URL: "${{ (inputs.env == 'prod' || github.event.schedule) && 'imagineyourdatacom-origin' || 'devimagineyourdatacom-origin' }}"
+  CLOUDFRONT: "${{ (inputs.env == 'prod' || github.event.schedule) && 'E2G3JHLVFJYNLC' || 'E1CPUDIUW43YST' }}"
+jobs:
+  build:
+    runs-on: [small, ui]
```

```suggestion
    runs-on: [edocs, big]
```
testing using edocs runners